akka_microservices/ibex35Microservice/src/main/scala/com/tapiporla/microservices/retrievers/common/Retriever.scala

```diff
@@ -0,0 +1,13 @@
+package com.tapiporla.microservices.retrievers.common
+
+import akka.actor.{Actor, ActorLogging}
+
+
+object Retriever {
+  object UpdateHistoricData //Used to keep data updated
+}
+
+
+trait Retriever extends Actor with ActorLogging {
```

Comment:
If the body is empty, I would remove braces.

Reply:
Done :)